cadence/samples/hello_activity.py

```diff
@@ -47,7 +48,7 @@ async def get_greeting(self, name):
     worker.register_workflow_implementation_type(GreetingWorkflowImpl)
     factory.start()
 
-    client = WorkflowClient.new_client(domain=DOMAIN)
+    client = WorkflowClient.new_client(domain=DOMAIN, protocol=PROTOCOL_GRPC, port=7833)
```

Comment:
this is having a mix of TChannel and gRPC. 
How about create another folder `samples-grpc` and then copy over the sample? gradually we will make all sample working in grpc only.

Reply:
Good call. This has been moved to a new file with the mix still, eventually when we migrate the rest of the methods it will be only grpc